packages/amplify-category-function/resources/awscloudformation/cloudformation-templates/lambda-cloudformation-template.json.ejs

```diff
@@ -139,7 +139,7 @@
                         }<% if (props.functionTemplate.parameters && props.functionTemplate.parameters.database && props.functionTemplate.parameters.database.resourceName) { %>,
                         {
                             "Effect": "Allow", 
-                            "Action": ["dynamodb:GetItem","dynamodb:Query","dynamodb:Scan","dynamodb:PutItem","dynamodb:UpdateItem","dynamodb:DeleteItem"],
+                            "Action": ["dynamodb:DescribeTable","dynamodb:GetItem","dynamodb:Query","dynamodb:Scan","dynamodb:PutItem","dynamodb:UpdateItem","dynamodb:DeleteItem"],
```

Comment:
Is this permission required for the template to work? If not, this PR shouldn't include a policy change

Reply:
Yes, the .NET SDK DDBv2 DocumentModel's Table class requires describeTable permission. Is there a way for a template (or template provider) to provide additional policies/statements to this?